docker-compose-starburst.yml

```diff
@@ -15,6 +15,45 @@ services:
     environment:
       POSTGRES_USER: dbt-trino
       POSTGRES_PASSWORD: dbt-trino
+    profiles:
+        - postgres
+
+  hive-metastore:
+    hostname: hive-metastore
+    image: 'michieldesmet/hive-metastore:latest'
+    ports:
+      - '9083:9083' # Metastore Thrift
+    volumes:
+      - ./docker/hive/metastore-site.xml:/opt/apache-hive-metastore-3.0.0-bin/conf/metastore-site.xml:ro
+    profiles:
+        - datalake
+  
+  minio:
+    hostname: minio
+    image: 'minio/minio:RELEASE.2022-05-26T05-48-41Z'
+    container_name: minio
+    ports:
+      - '9000:9000'
+      - '9001:9001'
+    environment:
+      MINIO_ACCESS_KEY: minio
+      MINIO_SECRET_KEY: minio123
+    command: server /data --console-address ":9001"
+    profiles:
+        - datalake
+
+  mc-job:
```

Comment:
what is `mc-job` and where it used?

Reply:
The minio client (`minio/mc`) creates the bucket required for the delta and iceberg connector to store its data.

Reply:
Please add a corresponding comment.

Reply:
`This job will create a bucket on minio` -> `This job will create "datalake" bucket on minio`